core/src/main/java/org/opensearch/sql/analysis/Analyzer.java

```diff
@@ -163,8 +177,12 @@ public LogicalPlan visitRelationSubquery(RelationSubquery node, AnalysisContext
   @Override
   public LogicalPlan visitTableFunction(TableFunction node, AnalysisContext context) {
     QualifiedName qualifiedName = node.getFunctionName();
+    Set<String> allowedCatalogNames = catalogService.getCatalogs()
```

Comment:
Is this `Set<String> getCatalogs()` interface still available?

Reply:
No, chained functions see one line below this line.